service/host.go

```diff
@@ -199,7 +206,7 @@ func (s *host) AddModules(modules ...ModuleCreateFunc) error {
 		if !s.supportsRole(mi.Roles...) {
 			s.Logger().Info(
 				"module will not be added due to selected roles",
-				"roles", mi.Roles,
+				zap.Any("roles", mi.Roles),
```

Comment:
no `zap.SliceAny`? Is there an overhead when we have to later check that it's a slice or are you going to just %v it?